ass.js

```diff
@@ -65,6 +83,7 @@ function startup() {
 		// Save the file information
 		let resourceId = generateId(resourceIdType, resourceIdSize, req.file.originalname);
 		data[resourceId.split('.')[0]] = req.file;
+		data[resourceId.split('.')[0]].private = true;
```

Comment:
Maybe make this a config option. By setting this to `true` by default, users sharing images & videos over Discord would need to edit the source prior to running.